package.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Would suggest to add the following:
`"scripts": {
    "cypress:open": "cypress open"
  }`
  
  Reference:
  https://docs.cypress.io/guides/getting-started/opening-the-app#cypress-open